src/content/docs/apm/agents/java-agent/instrumentation/instrument-browser-monitoring-java-agent-api.mdx

```diff
@@ -125,7 +128,6 @@ If your framework does not allow you to enable browser monitoring from our UI, w
        <cfoutput>#header#</cfoutput>
        ...
        ...
```

Comment:
@jtduffy did you really want two ellipses here? Could we remove one?

Reply:
One is fine.